id/standalone_group.go

```diff
@@ -29,8 +27,8 @@ type Group struct {
 
 // SealedGroup is an encrypted structure which contains data about a user group.
 type SealedGroup struct {
-	// The unique ID of the group.
-	GID uuid.UUID
+	// The group identifier.
+	GID string
```

Comment:
Well inside the string will always be a UUID since `NewUser`/`NewGroup` always creates it this way and there is no other way to create instances. If we type it to UUID, we will have to convert back and forth between UUID and string on a lot of method calls, and this operation is not necessarily cheap in terms of performance if you look at the implementation. This way we only convert once, after creating the UUID.

Reply:
I can change it, but I don't really see the advantages. What do you think?